schema.graphql

```diff
@@ -1,3 +1,5 @@
 type PureProxy @entity {
   id: ID! # pure proxied accountId
+  blockNumber: Int!
+  extrinsicIndex: String!
```

Comment:
It seems that extrinsicIndex is also better to be Int, as they should be of the same type to make it easier to work with it together